source/_docs/guides/composer_convert.md

```diff
@@ -0,0 +1,118 @@
+
+This will create a new directory based on the example project [pantheon-systems/example-drops-8-composer](https://github.com/pantheon-system/example-drops-8-composer) in the `my-site-composer` directory. The directory structure should now look like this;
+```sh
+Sites
+|-my-site
+|-my-site-composer
+```
+
+## Add in your Contrib/Custom code
+
+What makes your site code unique is your selection of contributed modules and themes and any custom modules or themes your development team has created. These customizations need to be replicated in our new project structure.
+
+### Contributed Code
+#### Modules
+A Composer managed site should be able to include all custom code via Composer. Begin by reviewing your existing site's code. Check for contributed modules in `/modules`, `/modules/contrib`, `/sites/all/modules`, and `/sites/all/modules/contrib`. Add these modules to your new codebase using Composer by doing the following for each module in the `my-site-composer` directory:
+```sh
+composer require drupal/module_name
+``` 
+Where `module_name` is the machine name of the module in question
+
+#### Themes
+Repeat the same process with themes, checking `/themes`, `/themes/contrib`, `/sites/all/themes`, and `/sites/all/themes/contrib`. Use Composer in the same way as above to require these.
+
+#### Libraries
+Libraries can be handled in the same way, the specifics dependent on how your library code was included in the source site. If using the Libraries API, you may have to do additional work to ensure the libraries function properly.
+
+### Custom Code
+#### Modules and Themes
+
+Custom code should be manually copied from the existing site repository.
+
+module:
+```sh
+me@my-computer:~/Sites/my-site(composify) $ cp -r /modules/custom/awesome_module ../my-site-composer/web/modules/custom
+```
+theme:
+```sh
+me@my-computer:~/Sites/my-site(composify) $ cp -r /themes/custom/great_theme ../my-site-composer/web/themes/custom
+```
+
+Follow suit with any other custom code you need to carry over.
+
+#### Settings.php
+Your existing site may have customizations to `settings.php` or any other config files. Review these carefully and extract relavent changes from these files to copy over, always review any file paths referenced in the code as these may change in the transition. It is not wise to completely overwrite the file with the old one as there are customizations for moving the configuration directory you don't want to overwrite.
+
+#### Configuration
+The preferred (and assumed) location of the configuration directories when using a nested docroot and Composer is at the root of the repository next to the web directory:
+```
+my-site-composer
+|-web
+|-config    <--Here!
+|-vendor
+|-composer.json
+|-etc...
+```
+
+Locate the configuration files in your existing site and move them here. If they are stored in the files directory on your existing site, you will want to retreive them via SFTP (as the Git clone would not contain them).
```

Comment:
Perhaps mention somewhere that the example project is already configured to put the config in this location.